samples/language/ReturnStatement.qs

```diff
@@ -21,6 +21,13 @@ namespace MyQuantumApp {
         // Any code following a return statement is inaccessible.
     }
 
+    function Returns42Implicit() : Int {
+        // If the last expression is not followed by a semicolon,
+        // its value is implicitly returned, in which case the return keyword is not necessary.
+        42 // This literal value is implicitly returned.
+        // Any code following an implicit return would cause the compilation to fail.
+    }
+
     function ReturnsUnit() : Unit {
```

Comment:
Probably, its worth having an example explicitly returning Unit here for completeness? And remove `samples/language/Unit.qs` example? I remember coming across that sample and thought it was mentioned here but it isnt. Thoughts?

```qsharp
function ReturnsUnitExplicit() : Unit {
        return ();
    }
 ```